Assets/Developers/Dennis/Components/Tags/BotTagComponent.cs

```diff
@@ -0,0 +1,8 @@
+﻿using UnityEngine;
+
+namespace TagComponents
```

Comment:
There's a bit of a debate on whether namespace in the same project should be a concept (`TagComponents`) or the project's name/indentifier (For example, in the framework the namespace was NUJAS). I usually lean toward namespacing using project's name, but @ico3939 can be the middle ground here.

Reply:
I've found that using the project name as the namespace is a bit easier for me to mentally organize, but I don't really have a preference in particular and am open to either.

Reply:
I guess we can just do Project name to limit hassle. This project is small enough to not care much about it anw.